tests/shell/include_common.sh.in

```diff
@@ -380,626 +345,29 @@ export_check()
 	fi
 
 	diff "$SPECEXPORT" "$SPECCHECK"
-	if [ $? != "0" ]
-	then
+	if [ $? != "0" ]; then
 		check_failed "$EXPORTS" "$TESTNAME" spec
 	fi
 
 	diff "$DIREXPORT" "$DIRCHECK"
-	if [ $? != "0" ]
-	then
+	if [ $? != "0" ]; then
 		check_failed "$EXPORTS" "$TESTNAME" dir
 	fi
 
 	diff "$USEREXPORT" "$USERCHECK"
-	if [ $? != "0" ]
-	then
+	if [ $? != "0" ]; then
 		check_failed "$EXPORTS" "$TESTNAME" user
 	fi
 
 	diff "$SYSTEMEXPORT" "$SYSTEMCHECK"
-	if [ $? != "0" ]
-	then
+	if [ $? != "0" ]; then
 		check_failed "$EXPORTS" "$TESTNAME" system
 	fi
 
 	diff "$MOUNTEXPORT" "$MOUNTCHECK"
-	if [ $? != "0" ]
-	then
+	if [ $? != "0" ]; then
 		check_failed "$EXPORTS" "$TESTNAME" mount
 	fi
 }
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
 # empty lines up to 1000 so that line numbers in the resulting scripts are more useful
```

Comment:
It seems like this was not done here.

The idea is that resulting line numbers simply need to be subtracted by 1000, which is much easier to do than to subtract with 373.

You might want to add comment lines, so that the code formatter does not removes them.

Reply:
The line after `@INCLUDE_COMMON@` in the original file, i.e. line 2, should now be line 1002 in the output file.